packages/pretty-format/src/index.js

```diff
@@ -397,20 +397,44 @@ const DEFAULTS: Options = {
 function validateOptions(opts: InitialOptions) {
   Object.keys(opts).forEach(key => {
     if (!DEFAULTS.hasOwnProperty(key)) {
-      throw new Error('prettyFormat: Invalid option: ' + key);
+      throw new Error(`pretty-format: Unknown option "${key}"`);
     }
   });
 
   if (opts.min && opts.indent !== undefined && opts.indent !== 0) {
-    throw new Error('prettyFormat: Cannot run with min option and indent');
+    throw new Error('pretty-format: Options "min" and "indent" cannot be used together');
   }
 }
 
+function normalizeTheme(themeOption: mixed) {
```

Comment:
Thanks for your patience while I learn Flow the hard way, especially how to balance typing at package interface for calling code that checks Flow type and coding to protect possible incorrect inputs from code that doesn’t.